nixos/modules/services/misc/sssd.nix

```diff
@@ -38,6 +42,15 @@ in {
           For this to work, the <literal>ssh</literal> SSS service must be enabled in the sssd configuration.
         '';
       };
+      environmentFile = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        description = ''
+          File containing environment variables to be passed to the sssd service,
```

Comment:
There's a good description format that's shared between a number of `environmentFile` options throughout NixOS:
- [services.hedgedoc.environmentFile](https://search.nixos.org/options?channel=21.05&from=0&size=50&sort=relevance&type=packages&query=services.hedgedoc.environmentFile)
- [services.prometheus.environmentFile](https://search.nixos.org/options?channel=21.05&show=services.prometheus.environmentFile&from=0&size=50&sort=relevance&type=packages&query=services.prometheus.environmentFile)
- [services.murmur.environmentFile](https://search.nixos.org/options?channel=21.05&show=services.murmur.environmentFile&from=0&size=50&sort=relevance&type=packages&query=services.murmur.environmentFile)
I'd advise to use it for this option too.

Reply:
Thanks for your input! That looks indeed quite useful. I'm going to update the description later (possibly tonight).

And it is good to know that #24288 exists. I wasn't aware of that :)